pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.black]
+line-length = 88
```

Comment:
So we are using 88 as the TVM standard column length for Python codes? I know this is the default and the suggested value in black so just want to double confirm.

Reply:
@tqchen confirmed this should be 100.